### PR TITLE
Add CONTRIBUTION.md

### PR DESCRIPTION
**Pull request by ManosMagnus**

### CODE REVIEW

File: CONTRIBUTION.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+If you have a question or problem to use weebots framework please ask on
+[deepbots group](https://groups.google.com/forum/#!forum/deepbots)
+
+## Found an issue?
+
+If you find an bug, please raise an issue in
+https://github.com/aidudezzz/deepbots/issues
+
+Please make sure that the bug comes from deepbots framework and not from
+weebots simulator.
```
Comment (tsampazk) [inline CONTRIBUTION.md:19]: Change to Webots

File: CONTRIBUTION.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+## Coding
+
+There is always lot of thinks to be done! Pull Request are always welcome. If
```
Comment (tsampazk) [inline CONTRIBUTION.md:23]: "a lot of *things*", Pull Request*s*

File: CONTRIBUTION.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+you are interesting in contributing a use case example please refer to
```
Comment (tsampazk) [inline CONTRIBUTION.md:24]: Interested, an example use case

File: CONTRIBUTION.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+[deepworlds](https://github.com/aidudezzz/deepworlds) repository.
+
+If you are interesting in code contribution please follow the guideline:
```
Comment (tsampazk) [inline CONTRIBUTION.md:27]: interested

File: CONTRIBUTION.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+- Pip install dev requirements
+
+  ```
+  pip install -r requirements/dev.txt
+  ```
+
+- We are using [pre-commit](https://pre-commit.com/) module for syntax and
+  style checking.
+- Please [rebase](https://git-scm.com/docs/git-rebase) before open a Pull
```
Comment (tsampazk) [inline CONTRIBUTION.md:48]: before opening a

File: CONTRIBUTION.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+  Request.
+
+- Use atomic commits for the PRs.
+
+- If you are interesting in implementing and new feature, please open a
+  tracking issue to make us known that you are working on it.
```
Comment (tsampazk) [inline CONTRIBUTION.md:54]: to make it known to us that

File: CONTRIBUTION.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+- Review to other PRs are always welcome.
```
Comment (tsampazk) [inline CONTRIBUTION.md:56]: Reviews

**Comment by ManosMagnus** [comment]
Done, ready for review

File: CONTRIBUTION.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+If you find an bug, please raise an issue in
+https://github.com/aidudezzz/deepbots/issues
+
+Please make sure that the bug comes from deepbots framework and not from Webots
+simulator.
+
+## Coding
+
+There is always a lot of things to be done! Pull Requests are always welcome.
+If you are interested in contributing an use case example please refer to
```
Comment (tsampazk) [inline CONTRIBUTION.md:24]: "*a* use case example" or "example use case"

**Pull request merged**